templates/CRM/common/civicrm.settings.php.template

```diff
@@ -365,6 +365,24 @@ if (!defined('CIVICRM_MAIL_SMARTY')) {
   define('CIVICRM_MAIL_SMARTY', 0);
 }
 
+/**
+ * Enabled this setting if you want to activate debug mode
+ * Useful when the font end is broken and cv can't work any more
+ * Debugging option value: 0 'no' or 1 'yes'
+ * Show backtrace option values: 0 'no' or 1 'yes'
+ * Environnement option values: Production / Staging / Development
+ * Asset cache option values: auto / 1 (active)  / 0 (disabled)
+ * ECMAScript Module Loader 'esm_loader' option values: auto / browser / shim-fast / shim-slow
+ * Fatal error Handler: set path and php class
+ */
+$civicrm_setting['domain']['debug_enabled'] = 0 ;
```

Comment:
I think these should be up in the section around line 288, and also commented out like those. (Maybe you did mean to have these in comments because otherwise line 382 below doesn't make sense.)